src/auxinfo/info.rs

```diff
@@ -41,6 +48,68 @@ impl AuxInfoPrivate {
     pub(crate) fn decryption_key(&self) -> &DecryptionKey {
         &self.decryption_key
     }
+
+    /// Convert private material into bytes.
+    ///
+    /// 🔒 This is intended for use by the calling application for secure
+    /// storage. The output of this function should be handled with care.
+    pub fn into_bytes(self) -> Vec<u8> {
+        // Format:
+        // AUXINFO_TAG | key_len in bytes | key
+        //             | ---8 bytes------ | --key_len bytes---
+
+        let key = self.decryption_key.into_bytes();
+        let key_len = key.len().to_le_bytes();
+
+        [AUXINFO_TAG, &key_len, &key].concat()
```

Comment:
note: This format is much messier and harder to parse (see below) compared to, say, putting these fields into a struct and using serde to serialize them. I think it's important to use a fixed-length format like this anyway, to enforce non-malleability of the ciphertext but I'm not totally sure that the other approach wouldn't work.